src/libDirectoryService/DirectoryService.h

```diff
@@ -75,6 +76,28 @@ struct PoWSolution {
   }
 };
 
+struct DSGuardUpdateStruct {
+  PubKey dsGuardPubkey;
```

Comment:
Shall we pre-pend the members with `m_`?

Reply:
Sure. To be consistent, we should also change `PoWSolution`

Reply:
Fix for DSGuardUpdateStruct